default/props.conf

```diff
@@ -1,25 +1,41 @@
-
 [postfix_syslog]
 # Note:  Extending Splunk default settings (See $SPLUNK_HOME/etc/system/default/props.conf)
-EXTRACT-queue_id    = postfix/\w+\[\d+\]:\s+(?<queue_id>[A-Fa-f0-9]{6,20}):
+
+# Extract the subject if enabled in Postfix's configuration
 REPORT-subject = postfix_subject
-# Extract to/from/message-id (without the '<>'s)
+# Extract to/from/message-id/helo (without the '<>'s)
 REPORT-angle_brackets = postfix_angle_brackets
-EXTRACT-reason = \((?<reason>[^)]*)\)
-EXTRACT-reject_reason = >: (?<reject_reason>[^;]*);
+
+EXTRACT-queue_id = postfix/\w+\[\d+\]:\s+(?<queue_id>[A-Fa-f0-9]{6,20}):
+EXTRACT-bounce = postfix/bounce\[\d+\]: [a-fA-F0-9]{6,20}: (?<bounce_reason>[^:]+): (?<bounce_queue_id>[a-fA-F0-9]{6,20})$
+EXTRACT-status_reject = postfix/smtpd\[\d+\]: (?:NOQUEUE|[A-Fa-f0-9]{6,20}): (?<status>reject):
+EXTRACT-reason = status=[^\s]+\s+\((?<reason>.*)\)$
+EXTRACT-reject_reason = : (?<reject_reason>[^;:]+);
 EXTRACT-dest = relay=(?<dest_host>[^\[ ,]+)\[(?<dest_ip>[^: \]]+)\](?::(?<dest_port>\d+))?
 EXTRACT-remote_queue = queued as (?<xref>[A-Fa-f0-9]+) in reason
-EXTRACT-status_code = status=\w+ \((?:host \S+ said:\s*)?(?<status_code>\d+)
+#EXTRACT-status_code = status=\w+ \((?:host \S+ said:\s*)?(?<status_code>\d+)
 EXTRACT-src-connect = (?:dis)?connect(?:ion after (?:HELO|CONNECT))? from (?:(?<src_host>[^\[]+)\[(?<src_ip>[\d.]+)|(?<src>.*))
+# Extration of the different delays (cf. http://logreporters.sourceforge.net/faq.html#percentiles)
+EXTRACT-delays = ^(?<time_before_queue>[^/]+)/(?<time_in_queue>[^/]+)/(?<time_connecting>[^/]+)/(?<time_transmitting>[^$]+)$ in delays
 
 # Rename fields for CIM compliance with the Email data model
-FIELDALIAS-queue = queue_id AS internal_message_id
-FIELDALIAS-process = pid AS process_id
-FIELDALIAS-tofrom = to AS recipient, from AS src_user
-FIELDALIAS-nrcpt = nrcpt as recipient_count
-# Don't extract 'src_host' if "unknown" (typical with reverse DNS disabled)
-EVAL-src = coalesece(src, nullif(src_host,"unknown"), src_ip)
-# FIELDALIAS-src = src_host AS src, src_ip AS src
+FIELDALIAS-status_code = dsn as status_code
+FIELDALIAS-protocol = proto as protocol
+FIELDALIAS-filter_action = reject_reason as filter_action
+FIELDALIAS-internal_message_id = queue_id AS internal_message_id
+FIELDALIAS-process_id = pid AS process_id
+FIELDALIAS-src_user = from as src_user
+FIELDALIAS-recipient = to as recipient
+FIELDALIAS-orig_recipient = orig_to as orig_recipient
+FIELDALIAS-recipient_count = nrcpt as recipient_count
 
 LOOKUP-consts = postfix_consts sourcetype OUTPUT protocol, vendor_product
 LOOKUP-actions = postfix_actions status OUTPUT action
+
+EVAL-src = src_host." (".src_ip.")"
+EVAL-dest = dest_host." (".dest_ip.")"
+# Email CIM requires the delay field to be in milliseconds
+EVAL-delay = delay*1000
+EVAL-product = "Mail Server"
```

Comment:
Trying to avoid using EVAL as much as possible due to the performance hit on searching on EVAL-based fields.   For static values, lookups are more performant.